ohsome_quality_api/topics/presets.yaml

```diff
@@ -164,6 +164,36 @@ cycleway:
     - core
     - bkg
 
+power_lines:
+  name: power_lines
+  description: >-
+      All linear OSM features referring to power lines.
+  endpoint: elements
+  aggregation_type: length
+  filter: >-
+    power=lines and geometry:line
```

Comment:
I think it should be power=line (without `s`). Also I would add power=minor_line as well.

`power in (line, minor_line) and geometry:line`